source/wp-content/themes/wporg-developer-2023/src/style/style.scss

```diff
@@ -435,7 +432,7 @@ pre {
 
 .wporg-developer-code-block {
 	$half_padding: calc(var(--wp--preset--spacing--10) / 2);
-	$border_radius: 2px;
+	$border_radius: var(--wp--custom--code--border--radius);
```

Comment:
This isn't inline code, rather it's the container, so I don't think it should use this var.